primitives/src/taiko/protocol_instance.rs

```diff
@@ -103,15 +104,27 @@ impl ProtocolInstance {
 
     // keccak256(abi.encode(tran, newInstance, prover, metaHash))
     pub fn hash(&self, evidence_type: EvidenceType) -> B256 {
+        println!("chain_id: {:?}", self.chain_id);
```

Comment:
These will appear in both `zkvm` and `sgx` guests, maybe need to remove

Reply:
removed!